app/src/main/res/layout/task_row.xml

```diff
@@ -10,17 +11,20 @@
     <Button
         android:id="@+id/task_title"
```

Comment:
Cirrus comments

Reply:
Unfortunately the required changes are messing up the UI (buttons are becoming transparent). What I suggest is keeping the style as-is for the demo, and modifying it next week.